### PR TITLE
adds boxes of bandages, and bandages to go in them

### PR DESCRIPTION
**Pull request by CoiledLamb**
## About The Pull Request
have you ever walked into medbay, picked up a suture, and asked yourself "how do i know how to stitch myself up?"
do you long for the simplicity of a bandage that can be popped on, no thought required?
well, look no further! now introducing, the DeForest First Aid Bandage!
image
each box of bandages holds five each, which are all one time use items. using one heals up to 25 brute damage from a single part of the body.
boxes of bandages will drain much faster than sutures, and are much less precise for healing specific increments of damage, but heal more in exchange. they are also worse at stopping blood loss, but not completely useless for it.

they can be found in a handful of places:
(some) high value medkits
in paramedic's pockets
some medbay lobbies
some cargo holds
all sec infirmaries
in goodies on the cargo console
maint loot
and nanomeds!

i considered making these replace brute kits, but those are used in a lot of old references, so it felt a bit passe. so i made a new item that is a bit more balanced, so they can be mapped in and added to vendors without much worry.
## Why It's Good For The Game
cute item, heals the people who wouldn't really be going into medbay anyways, and is fairly flavorful. if it's a bit too strong as is i can tweak the number, but the goal is to not be powerful enough to outshine sutures, while still offering somewhat effective healing for rough housing assistants and antags
## Changelog
:cl:
add: adds boxes of bandages, a quick healing item
/:cl:

### CODE REVIEW

**Comment by CoiledLamb** [comment]
drafting to uh. mapedit these in and figure out a good place for them

**Comment by carlarctg** [comment]
I love this concept so much!!

Definitely want/need a burn version.

File: code/game/objects/items/storage/boxes/medical_boxes.dm [hunk 1]
```diff
@@ -129,3 +129,42 @@
...
+	icon_state = "brutebox"
+	base_icon_state = "brutebox"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	drop_sound = 'sound/items/handling/matchbox_drop.ogg'
+	pickup_sound = 'sound/items/handling/matchbox_pickup.ogg'
+	illustration = null
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = ITEM_SLOT_BELT
+	custom_price = PAYCHECK_CREW * .75
```
Comment (carlarctg) [inline code/game/objects/items/storage/boxes/medical_boxes.dm:145]:
```suggestion
	custom_price = PAYCHECK_CREW * 0.75
```

Comment (CoiledLamb) [reply code/game/objects/items/storage/boxes/medical_boxes.dm:145]: 👍

**Comment by carlarctg** [comment]
It would be superb if these could be craftable. Something like gauze, 10u libital? Probably making a 'homemade' subtype like soap so it doesn't inherit the DeForest branding.

**Comment by CoiledLamb** [comment]
> It would be superb if these could be craftable. Something like gauze, 10u libital? Probably making a 'homemade' subtype like soap so it doesn't inherit the DeForest branding.

this is on the todo list, already made the box they come in craftable

**Comment by GuillaumePrata** [comment]
This is a superbly cute idea, but with these values I just see it being looted at roundstart by the first person that passes by, it seems too good for mid combat healing.

If balance with these values become a block for this PR, a version that doesn't heal damage but still stops the bleeding would stay cute and useful. Stops people from hoarding all of them, as you rarely get multiple bleed wounds, while helping with the bleed wounds that everyone dislikes.

**Comment by CoiledLamb** [comment]
> This is a superbly cute idea, but with these values I just see it being looted at roundstart by the first person that passes by, it seems too good for mid combat healing.
> 
> If balance with these values become a block for this PR, a version that doesn't heal damage but still stops the bleeding would stay cute and useful. Stops people from hoarding all of them, as you rarely get multiple bleed wounds, while helping with the bleed wounds that everyone dislikes.

I can continue to tweak the numbers, I've spent a fair bit of time adjusting them already, but yeag the idea is they're a bit better for combat.
I can also edit the mapped in ones to be single bandages rather than entire boxes.

**Comment by Pickle-Coding** [comment]
Not a fan of straight up healing brute. A bandage healing 25 brute sounds insane. I think it would be better if they stopped blood loss from bleed wounds, being effective for the lowest tier wounds. For the higher tier wounds, I guess it could only block a portion of the blood loss, or just get destroyed quickly from higher tier wounds so it's only a temporary solution.

Players should be rewarded for using a bandage on a cut, because that makes sense. They shouldn't be rewarded for putting a bandage on a bruise. That's just silly.

**Comment by carlarctg** [comment]
Yeah, what are we meant to believe here? That the bandage somehow just magically healed the wound? Because it had  healing chemicals in its synthesis? How silly. Boy I hope someone got fired for that blunder.


(TG is not serious)

**Comment by SomeoneYouProbablyKnow** [comment]
> I think it's a lot better if combat healing isn't restricted to powergamers who know how to make salacid/libital chem beakers in 20 seconds, actually.
> 
> All this does is even the playing field with those who can't/didn't prepare.

I disagree with that logic, people who routinely make healing chem mixes are a very small minority + medical players with nothing to do.

Most powergamers go for the most convenient low effort options, which is why every single one of them is carrying a pilfered white medkit. This tool will significantly increase the average healing avaliable to them.

Does that all matter? No. This PR is very cute and should be merged right now.

**Comment by CoiledLamb** [comment]
the fortunate thing about these is that they still take time to use, and aren't roundstart available in any medkits. they can only be obtained through cargo, med vendors, maint loot, and some static spawns like some medbay lobbies, as well as a few cargo warehouses

**Review by Wallemations — APPROVED** [review]

**Comment by HoodMilk** [comment]
Give the clown a box of bandaids for Patch Adams RP (the peak of gameplay)

**Comment by pilotmans** [comment]
> Not a fan of straight up healing brute. A bandage healing 25 brute sounds insane. I think it would be better if they stopped blood loss from bleed wounds, being effective for the lowest tier wounds. For the higher tier wounds, I guess it could only block a portion of the blood loss, or just get destroyed quickly from higher tier wounds so it's only a temporary solution.
> 
> Players should be rewarded for using a bandage on a cut, because that makes sense. They shouldn't be rewarded for putting a bandage on a bruise. That's just silly.

kinda like a chest seal but for bleeding?

**Comment by CoiledLamb** [comment]
doesn't gauze already fix bleed wounds?
it feels weird to give this the same functionality as an existing item that is barely used. I'll keep the heal as it is for now, unless a maint is not happy with it

**Comment by ChungusGamer666** [comment]
So we are back to bruise packs and ointment?
One must imagine sisyphus happy

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -474,3 +474,27 @@
...
+	name = "first aid bandage"
+	desc = "A DeForest brand bandage designed for basic first aid on blunt-force trauma."
+	icon_state = "bandage"
+	inhand_icon_state = "bandage"
+	novariants = TRUE
+	amount = 1
+	max_amount = 1
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	heal_brute = 25
```
Comment (ChungusGamer666) [inline code/game/objects/items/stacks/medical.dm:488]: what the FUCK

Comment (CoiledLamb) [reply code/game/objects/items/stacks/medical.dm:488]: ??

**Comment by CoiledLamb** [comment]
> So we are back to bruise packs and ointment? One must imagine sisyphus happy

ointment was always a thing, these are much more limited than bruise packs though. bruise packs are still in the game also, in space ruins

File: code/game/objects/items/storage/medkit.dm [hunk 3]
```diff
@@ -333,6 +335,7 @@
 		/obj/item/autosurgeon/syndicate/emaggedsurgerytoolset = 1,
 		/obj/item/reagent_containers/hypospray/combat/empty = 1,
 		/obj/item/storage/box/evilmeds = 1,
+		/obj/item/storage/box/bandages = 1,
```
Comment (san7890) [inline code/game/objects/items/storage/medkit.dm:338]: these are the same weight so can you alphabetize these

File: code/game/objects/items/storage/medkit.dm [hunk 4]
```diff
@@ -36,6 +36,7 @@
 		/obj/item/lighter,
 		/obj/item/storage/fancy/cigarettes,
 		/obj/item/storage/pill_bottle,
+		/obj/item/storage/box/bandages,
```
Comment (san7890) [inline code/game/objects/items/storage/medkit.dm:39]: alphabetize

File: code/game/objects/items/storage/medkit.dm [hunk 5]
```diff
@@ -296,6 +297,7 @@
 		/obj/item/stack/medical/gauze = 2,
 		/obj/item/reagent_containers/hypospray/medipen/atropine = 2,
 		/obj/item/reagent_containers/medigel/sterilizine = 1,
+		/obj/item/storage/box/bandages = 1,
 		/obj/item/surgical_drapes = 1,
 		/obj/item/scalpel = 1,
 		/obj/item/hemostat = 1,
```
Comment (san7890) [inline code/game/objects/items/storage/medkit.dm:303]: alphabetize anything within the weight

**Review by san7890 — CHANGES_REQUESTED** [review]
i'm not certain how i feel about it healing 25 brute (even if it is per limb), what's the comparison to that vs sutures

**Comment by CoiledLamb** [comment]
> i'm not certain how i feel about it healing 25 brute (even if it is per limb), what's the comparison to that vs sutures

sutures are 10 - can reduce it down to 20, the idea is that you can't actually get the full heal off a box unless you're using it on someone in crit, as it's much less precise. they also run out really quickly, so while you can get a lot of healing out of them, it's very unlikely, and if you do there's no chance of you doing it again without finding another box (which can be tricky!)

**Comment by san7890** [comment]
ah okay

**Pull request merged**